test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/CryptoTransferSuite.java

```diff
@@ -98,7 +98,36 @@ public boolean canRunAsync() {
 		return true;
 	}
 
-	private HapiApiSpec autoAssociationSuite() {
+	private HapiApiSpec autoAssociationsRollbackConsistentlyOnLaterMistake() {
+		final String tokenA = "tokenA";
+		final String treasury = "treasury";
+		final String toBeNonCompliant = "toBeNonCompliant";
+		final String toBeAutoAssociated = "toBeAutoAssociated";
+
+		return defaultHapiSpec("AutoAssociationSuite")
+				.given(
+						cryptoCreate(treasury)
+								.balance(ONE_HUNDRED_HBARS),
+						cryptoCreate(toBeAutoAssociated)
+								.balance(ONE_HBAR)
+								.maxAutomaticTokenAssociations(1),
```

Comment:
Are all of the token association properties tested elsewhere? All of these functions are still permitted (even though auto-associate won't work) right?